min-required.txt

```diff
@@ -1,6 +1,6 @@
-lxml==2.1.0  # >=1.3.4 should work
+lxml==3.5.0
```

Comment:
Okay, then you're fine - everything since wheezy (oldstable) has new enough libxml and libxslt.